lib/rspec-puppet/support.rb

```diff
@@ -242,9 +242,17 @@ def facts_hash(node)
         'fqdn' => node,
         'domain' => node.split('.', 2).last,
         'clientcert' => node,
```

Comment:
Can I just say I hate the `Style/TrailingCommaInHashLiteral` setting that's in this project? It creates bigger diffs than needed.

Reply:
Noted. Although if its a rubocop setting, then its probably a standard in all our modules and tools by now.